### PR TITLE
feat: Set fixed width for first columns with image on grids

### LINKED ISSUE #169: Reduce column width with checkboxes/avatars/icons
**Problem**
On pages with tables: we have auto-width set for first column with checkboxes/avatars/icons which results with unaligned tables (positions and candidates on Dashboard) and creating extra space when these elements (checkbox/avatar/icon) have fixed width.

CleanShot 14 11 2024 at 16-21

CleanShot 14 11 2024 at 16-19

**Solution**
Set fixed 64px width (32px max width for content within cell, 20px left-padding, 12px right-padding) for these columns, so that tables on 1 page are aligned and we don't create extra space.

### PR DESCRIPTION
**Pull request by Lyams**
Closes #169, #239.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

**Review by PrinceYauj — COMMENTED** [review]
1) Amgalan posted an ss of a tasks page, so, I believe, this page should also be updated. Candidate's and position's tasks tabs most likely should be updated as well
2) the icon is not aligned properly, is that ok? 
image
cc @amgalanbadmaev

**Comment by amgalanbadmaev** [comment]
Yes, tasks should be aligned too. All pages with grids that have first columns with checkboxes/avatars/icons.

Thanks for noticing icon unaligned. If it will be comfortable to fix in this PR, please do, otherwise please create low priority UX/UI issue (no need to put milestone).

File: app/assets/stylesheets/src/ats/_main.scss [hunk 1]
```diff
@@ -20,3 +20,13 @@
...
+
+tr {
+  td,
+  th {
+    &:first-child {
+      width: 64px;
+      text-align: center;
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (PrinceYauj) [reply app/assets/stylesheets/src/ats/_main.scss:32]: Also, for some reason, icon in positions grid is still not aligned 
image

[the remark this answers is not in the document]
Comment (PrinceYauj) [reply app/assets/stylesheets/src/ats/_main.scss:32]: Moreover, looks like candidate's files tab was affected
before the changes: 
image
after:
image

[the remark this answers is not in the document]
Comment (Lyams) [reply app/assets/stylesheets/src/ats/_main.scss:32]: I forgot about this grid! Thanks.

**Review by PrinceYauj — APPROVED** [review]

**Pull request merged**